state/txindex/kv/export_testing.go

```diff
@@ -0,0 +1,5 @@
+package kv
+
+func GetKeys(indexer *TxIndex) [][]byte {
```

Comment:
Same comment regarding this code being part of the production binary. OK if it is intended, else we are introducing a public function to access a private one in prod code.

Reply:
Same answer to [this](https://github.com/cometbft/cometbft/pull/1176#discussion_r1318581213)